pkg/exchange/okex/okexapi/client_test.go

```diff
@@ -0,0 +1,107 @@
+package okexapi
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/c9s/bbgo/pkg/testutil"
+)
+
+func getTestClientOrSkip(t *testing.T) *RestClient {
+	if b, _ := strconv.ParseBool(os.Getenv("CI")); b {
+		t.Skip("skip test for CI")
+	}
+
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, "OKEX")
+	if !ok {
+		t.SkipNow()
```

Comment:
When this case really happens, how do we know from the CI log? I mean, is there any way to remind us to configure the credential?

Reply:
Should print log to indicate credentials need to be added